.github/workflows/migrate.yml

```diff
@@ -47,4 +47,9 @@ jobs:
       - name: Apply all pending migrations to the database
         run: npm run migrate:deploy
         env:
-          DATABASE_URL: ${{ secrets.DATABASE_URL }}
\ No newline at end of file
+          DATABASE_URL: ${{ secrets.DATABASE_URL }}
+      
+      - name: Push changes to the database monogodb
+        run: npm run mongodb:push
+        env:
+          MONGO_DB_URL: ${{ secrets.MONGO_DB_URL }}
```

Comment:
como a gente vai usar essa url em outros repos, acho melhor add como organization secret

no caso a gente usa aq como database_url do repo pq as permissoes desse usuario sao diferentes! se no caso do mongo nao for, podemos deixar stg/prod envs nos secrets da organização e carregar os secrets de acordo com a env